src/toolbar/toolbar-group.jsx

```diff
@@ -197,7 +196,7 @@ const ToolbarGroup = React.createClass({
     }, this);
 
     return (
-      <div {...other} className={className} style={this.prepareStyles(styles.root, style)}>
+      <div {...other} className={className} style={prepareStyles(Object.assign({}, styles.root, style))}>
```

Comment:
well since it's gonna show up everywhere, why not just put it in the `prepareStyles` function in the first place. this way we won't have to make the assumption that the styles passed down to `prepareStyles` are safe to mutate. we'll just make them safe ourselves.